src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1057,9 +1056,6 @@ vector<TripleWithPropertyPath> Visitor::visit(
             addVisibleVariable(
                 var->getMatchingWordVariable(s.substr(0, s.size() - 1)));
           }
-        } else if (propertyPath->asString() == CONTAINS_ENTITY_PREDICATE) {
-          // addVisibleVariable(var->getTextScoreVariable());
-          addVisibleVariable(var->getScoreVariable());
```

Comment:
Is it really necessary to completely throw out the score, or can we keep it? I am not sure whether we have discussed this, but it has been a while.